airbyte-cdk/bulk/core/load/src/integrationTest/kotlin/io/airbyte/cdk/mock_integration_test/MockDestinationConfiguration.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.mock_integration_test
+
+import io.airbyte.cdk.command.ConfigurationJsonObjectBase
+import io.airbyte.cdk.command.DestinationConfiguration
+import io.airbyte.cdk.command.DestinationConfigurationFactory
+import io.micronaut.context.annotation.Factory
+import jakarta.inject.Singleton
+
+class MockDestinationConfiguration : DestinationConfiguration()
+
+@Singleton class MockDestinationSpecification : ConfigurationJsonObjectBase()
+
+@Singleton
+class MockDestinationConfigurationFactory :
+    DestinationConfigurationFactory<MockDestinationSpecification, MockDestinationConfiguration> {
+
+    override fun makeWithoutExceptionHandling(
+        pojo: MockDestinationSpecification
+    ): MockDestinationConfiguration {
+        return MockDestinationConfiguration()
+    }
+}
+
+@Factory
+class MockDestinationConfigurationProvider(private val config: DestinationConfiguration) {
```

Comment:
I will keep apologizing for this every time anyone writes a destination 😞

Reply:
the amount of scaffolding to wire through an empty pojo is pretty funny :P